### ISSUE TITLE
Why doesn't this regex rule work?

### ISSUE THREAD

**Issue by jimrs** [koekeishiya/yabai#871]
Hi, I'm trying to use rules to unmanage System Preferences, but it doesn't seem to be working (other rules work fine):
```
yabai -m rule --add app='Preferences' manage=off topmost=on
yabai -m rule --add title='Preferences' manage=off topmost=on
```
I tried restarting yabai of course, I tried 'Preferences$' but doesn't help.  `yabai -m query --windows` says this:
```
        "id":784,
	"pid":1491,
	"app":"System Preferences",
	"title":"System Preferences",
	"frame":{
		"x":845.0000,
		"y":32.0000,
		"w":668.0000,
		"h":598.0000
	},
	"level":0,
	"role":"AXWindow",
	"subrole":"AXStandardWindow",
	"movable":1,
	"resizable":0,
	"display":1,
	"space":2,
	"visible":1,
	"focused":0,
	"split":"vertical",
	"floating":0,
	"sticky":0,
	"minimized":0,
	"topmost":0,
	"opacity":1.0000,
	"shadow":1,
	"border":0,
	"stack-index":0,
	"zoom-parent":0,
	"zoom-fullscreen":0,
	"native-fullscreen":0
```
Thanks for hlep

**Comment by 24unix** [comment]
Why Preferences and not „System Preferences“?

**Comment by jimrs** [comment]
> Why Preferences and not „System Preferences“?

I got that from another issue #361 by The Man himself, it should keep afloat other "preferences" windows. Does it matter tho? The rule should still match, no?

**Comment by 24unix** [comment]
In that posting it is „Preferences$“.

Mind that Dollar sign. That is a wildcard.

**Comment by jimrs** [comment]
> In that posting it is „Preferences$“.
> 
> Mind that Dollar sign. That is a wildcard.

Thanks for the reply, I tried the dollar sign too but it didn't work either. To my knowledge, the $ represents end of string. But you actually got me thinking that maybe I should put a few wildcards (like a dot) before the "Preferences" string. I will try it.

**Comment by 24unix** [comment]
Oh, yes, your'e right, $ can be EOL, too. But then the examples doesn't make any sense :-(

**Comment by jimrs** [comment]
So I'm back here again to comment it still doesn't work. I tried all the following variants both in `app=` and `title=` without any success:
```
.*Preferences
.*Preferences.*
^.*Preferences.*
^.*Preferences.*$
```
Can anyone help please?

**Comment by jimrs** [comment]
l m a o
I removed the `topmost=on` part and it seems to work now. Is this argument now deprecated? Or is it a bug?

**Comment by koekeishiya** [comment]
`topmost=on` was deprecated in version 2.4.0, released 2020-03-01, and removed in version 3.0.0, released 2020-05-01.
Please use the documentation provided with your installed version or look at the appropriate tag for this repo:
https://github.com/koekeishiya/yabai/blob/master/doc/yabai.asciidoc

Changes are documented here:
https://github.com/koekeishiya/yabai/blob/master/CHANGELOG.md

You should now use `layer=above` instead of `topmost=on`.

**Comment by jimrs** [comment]
thnks bru
